Cargo.toml

```diff
@@ -33,6 +33,7 @@ futures = "0.1.14"
 gfx = "0.16"
 num_cpus = "1.2"
 genmesh = "0.4"
+image = "0.15"
```

Comment:
I don't really have an opinion towards either crate.  I was mostly just looking for a way to decode images in Rust, image was the first crate I found.  I totally didn't realize that we also had imagefmt which does the same thing but with fewer formats.  @alteous you said that image runs slowly in debug mode, can you elaborate on how you came across that information?  Have you used the image crate before?  Are there any pull requests I could send to the image crate in order to improve this situation?